cpp/daal/src/algorithms/k_nearest_neighbors/kdtree_knn_classification_model_impl.cpp

```diff
@@ -78,17 +78,15 @@ size_t Model::getNumberOfFeatures() const
 
 } // namespace interface1
 
-namespace interface2
+namespace interface3
 {
 services::Status Parameter::check() const
 {
-    // Inherited.
-    services::Status s = daal::algorithms::classifier::Parameter::check();
```

Comment:
what is the reason to delete supper check?

Reply:
Because here, we need relaxed check relative to inherited